[onExpand]: move onExpand from helpers to image&gallery plugin config (#1146)
wix-incubator/rich-content: docs/docs/plugins_api/PluginCustomization.md

| `componentDataDefaults` | Default component data                                                         | `{ config: { size: 'content', alignment: 'center' } }` | No           | editor               |
| `insertToolbars`        | Which toolbars to add insert button to                                         | `[TOOLBARS.FOOTER]`                                    | No           | editor               |

### Image Plugin Settings

| setting key             | description                                                                    | default value                                          | is required? | editor/viewer config |
| ----------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------------ | ------------ | -------------------- |

> | `onExpand`              | a function that called when the user expands an image                          | none                                                   | No           | viewer               |

Author: ```suggestion
| `onExpand`              | called when the user expands an image                          | none                                                   | No           | viewer               |
```

### Gallery Plugin Settings

| setting key             | description                                                                    | default value                                          | is required? | editor/viewer config |
| ----------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------------ | ------------ | -------------------- |

> | `onExpand`              | a function that called when the user expands an image in the gallery           | none                                                   | No           | viewer               |

Author: ```suggestion
| `onExpand`              | called when the user expands an item           | none                                                   | No           | viewer               |
```
